### PR TITLE
🍎  Update GraphQL QuickStart & CodeSandbox

### PR DESCRIPTION
**Pull request by nlarew**
## Pull Request Info

- Updates realm-web to the latest version (1.2.0) in the GraphQL example repo (https://github.com/mongodb-university/realm-graphql-apollo-react/commit/8a2ca7c788bbcaa17618a12f7dbb9dfbc2a7004c) 
- Update the CodeSandbox link to use the GitHub repo as the source of truth
- Mirror a helpful code comment from the GitHub source on this page

### Docs staging link (requires sign-in on MongoDB Corp SSO):

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/fruit/web/graphql-apollo-react/

### CODE REVIEW

File: source/includes/steps-graphql-apollo-react-setup.yaml [hunk 1]
```diff
@@ -76,6 +76,9 @@ content: |
      const client = new ApolloClient({
        link: new HttpLink({
          uri: graphql_url,
+         // We define a custom fetch handler for the Apollo client that lets us authenticate GraphQL requests.
```
Comment (ccho-mongodb) [inline source/includes/steps-graphql-apollo-react-setup.yaml:79]: Question: in this code sample, would it be better to maintain the same imperative tone throughout the comments or a descriptive one like this? Maybe they can coexist by differentiating them visually?

// Connect to your MongoDB Realm app 
// Get a valid Realm user access token to authenticate requests

vs

// The logged in user's access token might be stale,
// Refreshing custom data also refreshes the access token
(and these three lines)

File: source/web/graphql-apollo-react.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@ and integrates into your app with idiomatic React components and hooks.
    
    If you don't want to clone the repository, the demo application is also
    available in-browser in the :codesandbox:`Realm GraphQL CodeSandbox
-   <s/mongodb-realm-graphql-reactapollo-847sh?file=/src/index.js>`.
+   <s/github/mongodb-university/realm-graphql-apollo-react?file=/src/index.js>`.
```
Comment (ccho-mongodb) [inline source/web/graphql-apollo-react.txt:33]: Just a general informational question: I followed the codesandbox link and it couldn't fetch the dependencies. Do you know what I might be doing wrong?
```
←→1 of 3 errors on the page
SyntaxError
Could not fetch dependencies, please try again in a couple seconds: Unexpected token < in JSON at position 0

←→2 of 5 errors on the page
ReferenceError
keyEventHandler is not defined

×
←→3 of 7 errors on the page
ReferenceError
keyEventHandler is not defined

×
←→4 of 9 errors on the page
ReferenceError
keyEventHandler is not defined
...
```

**Review by ccho-mongodb — APPROVED** [review]
LGTM, addresses the task. Just have a couple questions.

File: source/web/graphql-apollo-react.txt (hunk 2 above)
Comment (nlarew) [reply source/web/graphql-apollo-react.txt:33]: Looking into this, spooky codesandbox errors 🤔

**Pull request merged**